google/cloud/logging_v2/client.py

```diff
@@ -352,9 +350,9 @@ def get_default_handler(self, **kw):
 
         if isinstance(monitored_resource, Resource):
```

Comment:
@daniel-sanche  is it possible to document the behavior of `get_default_handler` more thoroughly somewhere, including the default for each environment? I think some users will be initially confused why their logs are showing up in `stdout` and/or whichever other sinks they might be streaming their stdout logs to

Reply:
I added this suggestion to the [open issue](https://github.com/googleapis/python-logging/issues/245) around updating the docs for v3.

Let me know if you have thoughts about whether docs should be updated along with each change, or as a pass at the end of the milestone